packages/react-instantsearch/src/widgets/Panel.js

```diff
@@ -0,0 +1,37 @@
+import Panel from '../components/Panel.js';
+
+/**
+ * The Panel component will add a title to a widget. It will also handle when a wrapped widget doesn't have any refinement:
```

Comment:
As for now, I implemented it for widgets I thought it makes sense that's why I list them here. (but we could also have written a guide). 

I don't now if allowing something like below is pertinent: 

```js
<Panel>
    <HitsPerPage items=[]/>
</Panel>
```